edge-hub/test/Microsoft.Azure.Devices.Routing.Core.Test/query/ConditionVisitorTest.cs

```diff
@@ -11,6 +11,7 @@ namespace Microsoft.Azure.Devices.Routing.Core.Test.Query
     using Microsoft.Azure.Devices.Edge.Util.Test.Common;
     using Microsoft.Azure.Devices.Routing.Core.MessageSources;
     using Microsoft.Azure.Devices.Routing.Core.Query;
+    using Moq;
     using Xunit;
 
     [ExcludeFromCodeCoverage]
```

Comment:
Please remove [ExcludeFromCodeCoverage]

Reply:
See other reply.  Seems like this still applies, we don't want test code to affect code coverage numbers?

---
In reply to: [377464887](https://github.com/Azure/iotedge/pull/2469#discussion_r377464887) [](ancestors = 377464887)

Reply:
talked and resolved offline

---
In reply to: [377895001](https://github.com/Azure/iotedge/pull/2469#discussion_r377895001) [](ancestors = 377895001,377464887)